doc/devices/aer.rst

```diff
@@ -97,8 +70,7 @@ Noise models
 ~~~~~~~~~~~~
 
 One great feature of the ``'qiskit.aer'`` device is the ability to simulate noise. There are different noise models,
-which you can instantiate and apply to the device as follows
-(adapting `this <https://qiskit.org/documentation/apidoc/aer_noise.html>`_ qiskit tutorial):
```

Comment:
Link is gone, also don't know what exactly is being updated. Seems like this tutorial has either changed significantly or is just gone.

Reply:
Perhaps [this](https://qiskit.github.io/qiskit-aer/tutorials/3_building_noise_models.html) or [this](https://qiskit.github.io/qiskit-aer/tutorials/4_custom_gate_noise.html)?